WORKSPACE

```diff
@@ -32,6 +32,7 @@ protobuf_deps()
 # Their last and only one was on Sep 1, 2016. So, we need to pin to a
 # particular git commit, and this one was made on Feb 6, 2020.
 GOOGLEAPIS_GIT_SHA = "052b274138fce2be80f97b6dcb83ab343c7c8812"
+
```

Comment:
Any reason for this newline?

Reply:
Added automatically by my editor which runs `buildifier` on save.